locales/ko-KR.yml

```
@@ -829,6 +829,15 @@ reloadToApplySetting: "이 설정을 적용하려면 페이지를 새로고침
 needReloadToApply: "변경 사항은 새로고침하면 적용됩니다."
 showTitlebar: "타이틀 바를 표시하기"
 clearCache: "캐시 비우기"
+timelineCache: "타임라인 캐시"
+timelineCacheDescription: "내 타임라인, 리스트, 안테나의 캐시를 삭제합니다."
+purgeHomeTimelineCache: "홈 타임라인 캐시 삭제"
+purgeUserTimelineCache: "사용자 타임라인 캐시 삭제"
+selectUserList: "리스트 선택"
+purgeUserListTimelineCache: "리스트 타임라인 캐시 삭제"
+selectAntenna: "안테나 선택"
```

YAMLlint (1.37.1)

[error] duplication of key "selectAntenna" in mapping

(key-duplicates)